### ISSUE TITLE
wwwyzzerdd for the Web

### ISSUE THREAD

**Issue by teolemon** [derenrich/wwwyzzerdd#16]
I often find myself getting data from official websites about entities like this:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/1689815/170827902-56c58768-298a-44c8-b30d-2f820d08f438.png">

Extending wwwyzzerdd to a few common use-cases (like grepping social media accounts for an entity where we already have the official website) would be great.

**Comment by derenrich** [comment]
Yes this is something I want to do. It's a little tricky to do right because matching the webpage to the item isn't trivial.

There's actually a plugin that tries to do this already though. Maybe take a look at the similarly named [Wikidata for Web](https://www.wikidata.org/wiki/Wikidata:Tools/Wikidata_for_Web).
